packages/next-swc/crates/next-core/src/mode.rs

```diff
@@ -53,7 +53,7 @@ impl NextMode {
     pub fn minify_type(&self) -> MinifyType {
         match self {
             NextMode::Development => MinifyType::NoMinify,
-            NextMode::Build => MinifyType::Minify,
+            NextMode::Build => MinifyType::NoMinify,
```

Comment:
Guessing this was for debugging, needs to be reverted: 
```suggestion
            NextMode::Build => MinifyType::Minify,
```